.github/ISSUE_TEMPLATE/Questions.md

```diff
@@ -4,4 +4,13 @@ about: Ask questions about the library
 
 ---
 
+Learn more about how your form project could leverage the US Forms System. To help all of us use our time most efficiently:
 
+- **What is the scope of your project?**
+- **What are you using/planning on using to build your form?**
+- **What is the complexity?**

```

Comment:
Can we define "complexity" here? I'm not sure how I would answer this question myself.